FindPOLine/FindPOLine.js

```diff
@@ -58,7 +58,12 @@ const resultsFormatter = {
 
 const INIT_PAGINATION = { limit: PLUGIN_RESULT_COUNT_INCREMENT, offset: 0 };
 
-const FindPOLine = ({ addLines, isSingleSelect, ...rest }) => {
+const FindPOLine = ({
+  addLines,
+  crossTenant = false,
```

Comment:
THe default value can be moved to `FindPOLine.defaultProps`

Reply:
`defaultProps` is about to be deprecated in the next major version of the React https://github.com/folio-org/ui-receiving/pull/521